airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -113,6 +114,17 @@ def _open(self, binary):
             else:
                 uri = f"{storage}{user}@{host}:{port}/{url}"
             return smart_open.open(uri, transport_params=transport_params, mode=mode)
+        elif storage in ("https://"):
+            transport_params = None
+            if self._provider["user_agent"]:
+                airbyte_version = environ["AIRBYTE_VERSION"]
```

Comment:
Not sure if environment variables from the server are passed to source container.

Reply:
I have checked it and it is working. I get this in the logs:
airbyte-worker      | 2022-07-07 13:07:10 INFO i.a.w.i.DefaultAirbyteStreamFactory(internalLog):97 - TransportParams: {'headers': {'Accept-Encoding': 'identity', 'User-Agent': 'Airbyte/0.39.34-alpha'}}